package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-select",
-  "version": "1.2.1",
+  "version": "1.2.1-A",
```

Comment:
What's this mean? Why not bump the semver number?

Reply:
I don't want to conflict with react-select when they bump their number. That could cause confusion.

Reply:
As long as NPM handles this correctly in terms of when it comes time to upgrade.